src/jarabe/model/network.py

```diff
@@ -701,7 +701,7 @@ def network_hash(self):
         else:
             fl |= 1 << 6
 
-        hashstr = str(fl) + '@' + self.ssid.decode()
+        hashstr = str(fl) + '@' + self.ssid
```

Comment:
aa18879e9717dfe2d30f249549e9a43d6dd6da4f added this,   What makes you think self.ssid is a string?